es6.html

```diff
@@ -0,0 +1,116 @@
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings">Template strings</a></li>
+      </ul>
+
+      <p>You can view a more detailed list, including a comparison with other engines, on the <a href="https://kangax.github.io/compat-table/es6/">compat-table</a> project page.</p>
+    </div>
+    </div>
+
+    <div class="faq-item">
+      <h2 class="faq-title">
+         Which ES6 features are behind the --es_staging flag?
+      </h2>
+      <div class="faq-body">
+        <ul class="task-list">
+          <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class">Classes</a> (strict mode only)</li>
+          <li><a href="http://wiki.ecmascript.org/doku.php?id=harmony:object_literals">Object literal extensions</a></li>
+          <li>
+          <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol"><code>Symbol.toStringTag</code></a> (user-definable results for <code>Object.prototype.toString</code>)</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="faq-item">
+
+      <h2 class="faq-title">
+        I have my infrastructure set up to leverage the --harmony flag. Should I remove it?
+      </h2>
+      <div class="faq-body">
+        <p>The current behaviour of the <code>--harmony</code> flag on io.js is to enable <strong>staged</strong> features only. After all, it is now a synonym of <code>--es_staging</code>. As mentioned above, these are completed features that have not been considered stable yet. If you want to play safe, especially on production environments, consider removing this runtime flag until it ships by default on V8 and, consequently, on io.js. If you keep this enabled, you should be prepared for further io.js upgrades to break your code if V8 changes their semantics to more closely follow the standard.</p>
+      </div>
+    </div>
+
+    <div class="faq-item">
+
+      <h2 class="faq-title">
+         How do I find which version of V8 ships with a particular version of io.js?
+      </h2>
+      <div class="faq-body">
+        <p>io.js provides a simple way to list all dependencies and respective versions that ship with a specific binary through the <code>process</code> global object. In case of the V8 engine, type the following in your terminal to retrieve its version:</p>
+        <div class="highlight highlight-sh"><pre>iojs -e <span class="pl-s1"><span class="pl-pds">'</span>console.log(process.versions.v8)<span class="pl-pds">'</span></span></pre></div>
+
+        <p>E.g. output: <code>3.31.71.4</code>.</p>
+
+      </div>
+    </div>
+
+    <div class="faq-item">
+
+      <h2 class="faq-title">
+         How frequently do you plan on updating V8?
+      </h2>
+      <div class="faq-body">
+        <p>A formal plan has not been discussed yet but the TC is looking into regular updates, especially if there are no breaking changes from V8's API.</p>
```

Comment:
let's cut the section about v8 updates. that process is under active
discussion.

On Tuesday, January 13, 2015, Jeremiah Senkpiel <notifications@github.com>
wrote:

> In es6.html
> <https://github.com/iojs/iojs.github.io/pull/27#discussion-diff-22864727>:
>
> > +      <div class="faq-body">
> > +        <p>io.js provides a simple way to list all dependencies and respective versions that ship with a specific binary through the <code>process</code> global object. In case of the V8 engine, type the following in your terminal to retrieve its version:</p>
> > +        <div class="highlight highlight-sh"><pre>iojs -e <span class="pl-s1"><span class="pl-pds">'</span>console.log(process.versions.v8)<span class="pl-pds">'</span></span></pre></div>
> > +
> > +        <p>E.g. output: <code>3.31.71.4</code>.</p>
> > +
> > +      </div>
> > +    </div>
> > +
> > +    <div class="faq-item">
> > +
> > +      <h2 class="faq-title">
> > +         How frequently do you plan on updating V8?
> > +      </h2>
> > +      <div class="faq-body">
> > +        <p>A formal plan has not been discussed yet but the TC is looking into regular updates, especially if there are no breaking changes from V8's API.</p>
>
> cc @bnoordhuis <https://github.com/bnoordhuis>, what was the exact plan
> on when to do v8 updates?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iojs/iojs.github.io/pull/27/files#r22864727>.
>